agents/check_mk_agent.macosx

```diff
@@ -234,13 +266,6 @@ if [ -r "$MK_CONFDIR/fileinfo.cfg" ] ; then
     IFS=$OLD_IFS
 fi
 
-# Doesn't work since 10.14
```

Comment:
Deleting commented out code

Reply:
I decided to base the check on `sw_vers` output instead now: https://github.com/ThomasKaiser/Check_MK/commit/97c0280ce1473653f3f3e970736b412d98478b64